### ISSUE TITLE
Imports from wagtail generate deprecation warnings

### ISSUE THREAD

**Issue by banagale** [APSL/puput#272]
After installing puput, I got back a variety of `RemovedInWagtail50Warning` deprecation warnings.

These all refer to the wagtail 3.0 release notes [changes to module paths section](https://docs.wagtail.org/en/stable/releases/3.0.html#changes-to-module-paths).

It appears puput could use updates to reflect these changes across the app.

If this sounds right, please let me know I'd consider providing a PR for this update.

**Comment by CVYY39** [comment]
I get those too.

**Comment by banagale** [comment]
Okay, it would be good to know the status of #271 before starting in on a PR for this.

**Comment by csalom** [comment]
We're currently working in a new PR that faces this warnings #277
